frontend/src/components/object/ObjectTable.vue

```diff
@@ -175,7 +257,10 @@ const filters = ref({
         body-class="truncate"
       >
         <template #body="{ data }">
-          <div v-tooltip.bottom="{ value: data.name }">
+          <div
+            v-tooltip.bottom="{ value: data.name }"
+            :data-objectid="data.id"
```

Comment:
feels a bit precarious adding this attribute to the tooltip. maybe it should appear in a hidden column.
i wonder if it's also available from the dataKey for each row (https://primevue.org/datatable/#api.datatable.props.dataKey)